bedrock/firefox/templates/firefox/firstrun/new-firstrun.html

```diff
@@ -12,6 +12,8 @@
   {% javascript 'firefox_new_firstrun' %}
 {% endblock %}
 
+{% block global_nav %}{% endblock %}
+
```

Comment:
[Reply in the bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1354710#c18) is that what we've done here should satisfy the requirements OK.